src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +26,17 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_IsReflexive_BHamrick() {
```

Comment:
Good test but does not follow convention, which would be: `equals_True_Reflexive`.

My adding students' names last time was not conventional.

Reply:
Reworked.